openstack/cinder/templates/vcenter_datacenter_cinder_deployment.yaml

```diff
@@ -44,6 +44,7 @@ template: |
         hostname: cinder-volume-vmware-{= name =}
 {{ tuple "{= availability_zone =}" | include "kubernetes_pod_az_affinity" | indent 8 }}
 {{ include "utils.proxysql.pod_settings" . | indent 8 }}
+{{- include "kubernetes_maintenance_affinity" . | indent 4 }}
```

Comment:
This needs to go one line up, because it extends the affinity dict created by `include "kubernetes_pod_az_affinity"`